nix/preset/nix.nix

```diff
@@ -39,7 +39,7 @@
     commands = lib.mkOrder 300 [
       {
         type = "shell";
-        runtimeInputs = [pkgs.nix];
+        runtimeInputs = [pkgs.nix pkgs.coreutils];
```

Comment:
Why? The bash preset already does this and is enabled [by default](https://github.com/input-output-hk/tullia/blob/575362c2244498e8d2c97f72861510fa72e75d44/nix/module.nix#L185).

Reply:
Reproduced by `tullia run nix-preset`

I don't know why you commented out the PAHT in preset.bash.env.

Reply:
Sorry, this change was actually correct! I didn't read your changes properly